keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md

```diff
@@ -77,58 +139,15 @@ This sample should work together with [azure-spring-boot-sample-keyvault-certifi
    source script/setup.sh
    ```
 
-#### Using TLS with managed identity
-1. Replace the `restTemplateWithTLS` bean in `SampleApplicationConfiguration.java` as
-    <!-- embedme ../azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/src/main/java/com/azure/spring/security/keyvault/KeyVaultJcaManagedIdentitySample.java#L22-L40 -->
+#### Using TLS/mTLS with managed identity
+1. Replace the `CREDENTIALTYPE` value in `RestTemplateConfiguration.java` as
```

Comment:
How about `WebClientConfiguration.java`. I mean use managed identity in WebClient.

Reply:
For now, the doc is correct. I know it's better to use a common/util class to avoid misunderstanding.